### ISSUE TITLE
Apple ARM gfortran should be updated

### ISSUE THREAD

**Issue by Nicholaswogan** [conda-forge/ctng-compiler-activation-feedstock#82]
Apple ARM gfortran version 11.3 and 12.1 are released, yet conda only supplies v11.0.1.dev0.

**Comment by rgommers** [comment]
It's now at `11.3.0`, but all the non-macOS packages are at 12.2.0 already. And 11.3.0 does not seem to work on macOS 13:
```
Sanity check compile stderr:                                    
arm64-apple-darwin20.0.0-gfortran: warning: could not understand version ‘13.00.00’           
ld: -rpath can only be used when targeting Mac OS X 10.5 or later
collect2: error: ld returned 1 exit status
```

Does anyone know if there is a way to get the current build to work on macOS 13?

**Comment by Nicholaswogan** [comment]
You might try completely re-installing command line tools. But ya, it would be nice if conda was updated to gfortran 12

**Comment by rgommers** [comment]
Reinstalling command-line tools completely doesn't fix it. Reinstalling is done by (1) moving XCode.app to the bin, (2) `sudo rm -rf /Library/Developer/CommandLineTools`, (3) reinstalling command line tools via the popup macOS gives you after, and (4) opening a fresh terminal.

Using a distutils build of SciPy turns up a second issue, which boils down to compiler detection during the build choking on this warning:
```
% gfortran -dumpversion
gfortran: warning: could not understand version ‘13.00.00’
11.3.0
```
